demisto_sdk/commands/lint/commands_builder.py

```diff
@@ -72,7 +58,7 @@ def build_bandit_command(files: List[Path]) -> str:
     Returns:
         str: bandit command
     """
-    command = "python3 -m bandit"
+    command = "bandit"
```

Comment:
```suggestion
    command = "python -m bandit"
```
in this way, we will be able to avoid `PATH` issues.

Reply:
When using simply `flake8`, `bandit` or `pylint` we guaranteed that it will works every time. When running from virtual environment, we install those packages therefore we will run them on the venv itself.
When running from docker, we will use either python2 command or python3 command, based on the python version inside the docker.
Another thing is that we have no guarantee that `python` alias will work - in the current content env there is no `python`, only `python3` or `python2`.